web/apps/atelier-des-idees/src/components/Select/index.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import SelectComponent, { components } from 'react-select';
+
+const DropdownIndicator = props =>
+    components.DropdownIndicator && (
+        <components.DropdownIndicator {...props}>
+            <span className="dropdown" />
+        </components.DropdownIndicator>
+    );
+
+const MultiValueLabel = props => <components.MultiValueLabel {...props} />;
+
+const MultiValueRemove = props => <components.MultiValueRemove {...props} />;
+
+class Select extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            selectedOption: null,
+        };
+        this.handleChange = this.handleChange.bind(this);
+    }
+
+    handleChange(selectedOption) {
+        this.setState({ selectedOption });
+        this.props.isMulti
```

Comment:
Déterminer la valeur à envoyer dans une const et appeler `this.props.onSelected` avec cette const